.github/workflows/spec.yml

```diff
@@ -0,0 +1,38 @@
+name: "Static & Spec Tests"
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  StaticAndSpecTests:
+    name: "Static & Spec Tests (Ruby ${{ matrix.ruby_version }})"
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby_version: ['2.5', '2.7']
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout Source
+      uses: actions/checkout@v2
+
+    - name: "Activate Ruby ${{ matrix.ruby_version }}"
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby_version }}
+        bundler-cache: true
+        without: "development"
```

Comment:
there's a warning regarding `without` in the output https://github.com/puppetlabs/pdk/pull/1030/checks#step:3:1 so this may not be doing what is expected.

Reply:
You'll want to add this env declaration:

```
    env:
      BUNDLE_JOBS: 4
      BUNDLE_WITHOUT: "development"
```